pkg/bazel/workspace/finder_test.go

```diff
@@ -6,38 +6,59 @@
  * Full License text is in the LICENSE file included in the root of this repository.
  */
 
-package pathutils
+package workspace
 
 import (
 	"fmt"
 	"io/fs"
 	"os"
 	"testing"
 
+	stdlib_mock "aspect.build/cli/pkg/stdlib/mock"
+
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/gomega"
-
-	stdlib_mock "aspect.build/cli/pkg/stdlib/mock"
```

Comment:
Revert?

Reply:
I beat you to it: https://github.com/aspect-build/aspect-cli/pull/219/commits/ed5d20a468ac27ea70d44772d685f85b7f57fcf6.